src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java

```diff
@@ -33,6 +34,11 @@ public class HoppityEggsConfig {
     @FeatureToggle
     public boolean sharedWaypoints = true;
 
+    @Expose
+    @ConfigOption(name = "Adjust player opacity", desc = "Adjust the opacity of players near shared & guessed egg waypoints.")
+    @ConfigEditorSlider(minValue = 0, maxValue = 255, minStep = 1)
+    public int playerOpacity = 100;
```

Comment:
might be more user friendly for this to be from 1-100, and convert that later on into 1-255 so people can look at it more as a percentage

Reply:
That is true, done!